agent/src/main/java/com/criteo/hadoop/garmadon/agent/tracers/hadoop/hdfs/FileSystemTracer.java

```diff
@@ -42,7 +43,7 @@ public static void initEventHandler(Consumer<Object> eventConsumer) {
     public static class DeleteTracer extends MethodTracer {
 
         @Override
-        public ElementMatcher<? super TypeDescription> typeMatcher() {
+        protected ElementMatcher<? super TypeDescription> typeMatcher() {
```

Comment:
plz don't change visibility in front of all methods

Reply:
or this one I can in fact not change visibility